gen/src/generate/mod.rs

```diff
@@ -17,3 +17,11 @@ pub mod ucd;
 /// The standard auto-generated source disclaimer.
 const PREAMBLE: &'static str = "// WARNING: Auto-generated by the `unic-gen` crate.\n\
                                 // WARNING: DO NOT EDIT MANUALLY!";
+
+fn capitalize(str: &str) -> String {
```

Comment:
This reminds me that we may also want the same logic for `FromStr`, since, by UAX#44 definition, all property/value names are case-insensitive. Or, we can implement a `find()` function or something like that, that does more extensive search for value, so that we keep `FromStr` performant. What do you think?